amy/extrequests/templatetags/request_membership.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
+
+from django import template
+from django.utils.safestring import mark_safe
+
+from workshops.models import Membership
+
+register = template.Library()
+
+
+@register.simple_tag
+def membership_description(membership: Membership):
```

Comment:
My suggestion is to turn this tag into a Django template. There's a lot of template generation going on compared to "difficult template logic".

Here, the logic that may be difficult or "ugly" to achieve in the Django template is the calculation of alert type. I suggest to use a template tag to determine if this message is just informational or a warning.